scratch_kotlin_lib/src/main/java/dev/sgrem/scratch_kotlin_lib/Week4Homework.kt

```diff
@@ -0,0 +1,160 @@
+package dev.sgrem.scratch_kotlin_lib
+
+fun main() {
+
+    // 1. Task Data:
+    data class Task(
+        val title: String,
+        // var priority: Int = 1,
+        var priority: Priority = Priority.HIGH,
+        var completed: Boolean = false
```

Comment:
Small nit: you can have the last property (and params in a multi-line function declaration!) trailed with a comma.  That way, if you ever add new properties in the future, you won't have to edit this line and add the comma at that time.

```suggestion
        var completed: Boolean = false,
```

Note: This is also up to personal preference and can be quite subjective, but I like it 😆